src/goPackages.ts

```diff
@@ -238,7 +238,7 @@ export function isAllowToImportPackage(toDirPath: string, currentWorkspace: stri
 	let internalPkgFound = pkgPath.match(/\/internal\/|\/internal$/);
 	if (internalPkgFound) {
 		let rootProjectForInternalPkg = path.join(currentWorkspace, pkgPath.substr(0, internalPkgFound.index));
-		return toDirPath.startsWith(rootProjectForInternalPkg);
+		return toDirPath.startsWith(rootProjectForInternalPkg + '/') || toDirPath === rootProjectForInternalPkg;
```

Comment:
Any chance in Windows this would be `\` instead?

Reply:
Not sure... it based on pkg. It will fail on `let internalPkgFound = pkgPath.match(/\/internal\/|\/internal$/);` also..
Need Windows machine

Reply:
I see.. will push latest changes